packages/wonder-blocks-form/src/util/styles.ts

```diff
@@ -176,7 +176,7 @@ export const colorStates: StyleMap = {
             default: {
                 rest: {
                     border: semanticColor.input.checked.background,
-                    background: semanticColor.core.border.inverse.strong,
+                    background: semanticColor.core.border.knockout.default, // TODO(WB-2050): use white background
```

Comment:
In one of the next PRs, there'll be `background.base.default` which is `white`. Will update to that new token later on so we don't have to use a border token for a background here!